### ISSUE TITLE
Porsche 911 992 GT3R 

### ISSUE THREAD

**Issue by Kedd777** [thogue95/irFFB#18]
Hello good sir!

Could you add support for the new Porsche 911 992 GT3R pretty please?

**Comment by thogue95** [comment]
irFFB2022 does not use a list of cars like the original version.  All cars should have all features.  Are you saying it is not working?

**Comment by Kedd777** [comment]
Yes it's not working. But it also doesn't work on the old 911 GT3R either, not sure why. I also tried fuzzwah's version and it works with the old 911 but not the new one (the list is not updated).

**Comment by RenatoCoicev** [comment]
From my experience, "Understeer Wheel Force = 0" results in no understeer effect. Any number below 1 seems to have the same result, so no 0.4 or 0.8. Setting "Understeer Wheel Force = 1" brings the effect to life.

Because we are at it, I'd love to see zero becoming a valid wheel force. I struggle A LOT to identify understeer, even under the most aggressive configs. Zero wheel force might be able to help me out here.
